sycl/doc/SYCLRuntimeRFC.md

```diff
@@ -0,0 +1,97 @@
+scourse topic details
+
+- Category: [Runtimes](https://discourse.llvm.org/c/runtimes/64)
+- Title: "RFC: SYCL runtime upstreaming"
+- Tags: sycl
+
+## Overview
+
+Our SYCL runtime provides an implementation of the SYCL 2020 API specification.
+It's responsible for managing resources for enqueuing tasks to the offload
+device, tracking dependencies between them, and data movement between the host
+and devices. The SYCL runtime is device-agnostic and uses Unified Runtime
+(https://github.com/oneapi-src/unified-runtime) as an external dependency that
+serves as an interface layer between the SYCL runtime and device-specific
+backends. Unified Runtime has several adapters that bind to various backends in
+a similar fashion to libomptarget.
+
+This RFC contains a brief overview of the core functionality of the SYCL runtime
```

Comment:
Is there any specific question/comment that we will need to ask the community here?